README.md

```diff
@@ -1,29 +1,18 @@
-# Mantis Faucet Web
+# SC_EVM Faucet
 
-Web interface for Mantis Faucet
```

Comment:
Web faucet for EVM sidechain

Reply:
@NeilBurgess42 we shall explain this is a web interface of the faucet for EVM sidechain in the initial README

Reply:
OK